roles/edpm_ovs/tasks/user.yml

```diff
@@ -5,6 +5,7 @@
   vars:
     edpm_users_users:
       # 42476 is matching with the uid and gid created by kolla in the openstack containers
+      - {"name": "hugetlbfs", "gid": "42477", "group_only": true}
```

Comment:
this change lgtm but is not being picked up in the test run (and I'm trying to understand why... zuul thinks it is checked out correctly, but it is not being used by the create users & groups tasks:

```
TASK [osp.edpm.edpm_users : Create host users and groups] **********************
[1;30mtask path: /usr/share/ansible/collections/ansible_collections/osp/edpm/roles/edpm_users/tasks/main.yml:17[0m
[0;36mincluded: /usr/share/ansible/collections/ansible_collections/osp/edpm/roles/edpm_users/tasks/create_users_and_groups.yml for compute-0 => (item={'name': 'openvswitch', 'uid': '42476', 'gid': '42476', 'shell': '/sbin/nologin', 'groups': 'hugetlbfs', 'comment': 'openvswitch user'})[0m
```

i see we have this in the role default https://github.com/openstack-k8s-operators/edpm-ansible/blob/c8ed1ac822813bfe27d42cd98573ab93cb25b68d/roles/edpm_users/defaults/main.yml#L26-L28 

```
edpm_users_users:
  - {"name": "qemu", "uid": "107", "gid": "107", "shell": "/sbin/nologin", "comment": "qemu user"}
  - {"name": "hugetlbfs", "gid": "42477", "group_only": true}
```

so what you're adding here looks correct eg the group id and since the openvswitch user needs that group to exist it makes sense that we'd create it here (it seems like that is a requirement and should have been included here earlier?)

Reply:
This is not required. The issue was something else, fixed in https://github.com/openstack-k8s-operators/ci-framework/pull/1082

Reply:
oh @rabi it looks like this merged anyway after my review 

do we need to revert it then? I don't think it should hurt?

Reply:
as discussed in slack @lewisdenny will revert this